tests/endpoints/test_reminders.py

```diff
@@ -0,0 +1,272 @@
+import unittest
+from datetime import datetime
+from unittest.mock import Mock
+
+from fastapi.testclient import TestClient
+
+from api.core.settings import settings
+from api.endpoints.dependencies.database import create_database_session
+from api.endpoints.reminder.reminder_schemas import ReminderResponse
+from api.main import app
+
+client = TestClient(app)
+
+AUTH_HEADER = {"Authorization": f"Bearer {settings.auth_token}"}
+
+
+class TestUnauthedReminderAPI:
```

Comment:
It would be better if these inherit `unittest.Testcase` and use its methods rather than defining a dead `case = unittest.TestCase()` and using it. Later, you could also remove the `Test` prefix from this.

Reply:
I added this so that I can stick with the general pytest syntax, it integrates pretty well. The 'Test' in the name is a pytest convention.